modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/rest/resource/releases/SW360Release.java

```diff
@@ -129,19 +130,34 @@ public void setDownloadurl(String downloadurl) {
         this.downloadurl = downloadurl;
     }
 
-
     @JsonIgnore
-    public Map<String, String> getCoordinates() {
+    public Set<String> getPurls() {
```

Comment:
I think it would be more appropriate to use the PackageURL class instead of String to handle the object and only use the String representation when transfering it to SW360.

Reply:
I think that the `SW360Release` should be as near as possible to the representation which will be transferred and therefore I think that it is perfectly fine to have them as string. Especially since they are stored in the `externalId` map.